examples/Makefile.Android

```diff
@@ -159,92 +158,164 @@ all: create_temp_project_dirs \
 
 # Create required temp directories for APK building
 create_temp_project_dirs:
-	if not exist $(PROJECT_BUILD_PATH) mkdir $(PROJECT_BUILD_PATH) 
-	if not exist $(PROJECT_BUILD_PATH)\obj mkdir $(PROJECT_BUILD_PATH)\obj
-	if not exist $(PROJECT_BUILD_PATH)\src mkdir $(PROJECT_BUILD_PATH)\src
-	if not exist $(PROJECT_BUILD_PATH)\src\com mkdir $(PROJECT_BUILD_PATH)\src\com
-	if not exist $(PROJECT_BUILD_PATH)\src\com\$(APP_COMPANY_NAME) mkdir $(PROJECT_BUILD_PATH)\src\com\$(APP_COMPANY_NAME)
-	if not exist $(PROJECT_BUILD_PATH)\src\com\$(APP_COMPANY_NAME)\$(APP_PRODUCT_NAME) mkdir $(PROJECT_BUILD_PATH)\src\com\$(APP_COMPANY_NAME)\$(APP_PRODUCT_NAME)
-	if not exist $(PROJECT_BUILD_PATH)\lib mkdir $(PROJECT_BUILD_PATH)\lib
-	if not exist $(PROJECT_BUILD_PATH)\lib\$(ANDROID_ARCH_NAME) mkdir $(PROJECT_BUILD_PATH)\lib\$(ANDROID_ARCH_NAME)
-	if not exist $(PROJECT_BUILD_PATH)\bin mkdir $(PROJECT_BUILD_PATH)\bin
-	if not exist $(PROJECT_BUILD_PATH)\res mkdir $(PROJECT_BUILD_PATH)\res
-	if not exist $(PROJECT_BUILD_PATH)\res\drawable-ldpi mkdir $(PROJECT_BUILD_PATH)\res\drawable-ldpi
-	if not exist $(PROJECT_BUILD_PATH)\res\drawable-mdpi mkdir $(PROJECT_BUILD_PATH)\res\drawable-mdpi
-	if not exist $(PROJECT_BUILD_PATH)\res\drawable-hdpi mkdir $(PROJECT_BUILD_PATH)\res\drawable-hdpi
-	if not exist $(PROJECT_BUILD_PATH)\res\values mkdir $(PROJECT_BUILD_PATH)\res\values
-	if not exist $(PROJECT_BUILD_PATH)\assets mkdir $(PROJECT_BUILD_PATH)\assets
-	if not exist $(PROJECT_BUILD_PATH)\assets\$(PROJECT_RESOURCES_PATH) mkdir $(PROJECT_BUILD_PATH)\assets\$(PROJECT_RESOURCES_PATH)
-	if not exist $(PROJECT_BUILD_PATH)\obj\screens mkdir $(PROJECT_BUILD_PATH)\obj\screens
+	mkdir -p $(PROJECT_BUILD_PATH) 
+	mkdir -p $(PROJECT_BUILD_PATH)/obj
+	mkdir -p $(PROJECT_BUILD_PATH)/obj/src
+	mkdir -p $(PROJECT_BUILD_PATH)/src
+	mkdir -p $(PROJECT_BUILD_PATH)/src/com
+	mkdir -p $(PROJECT_BUILD_PATH)/src/com/$(APP_COMPANY_NAME)
+	mkdir -p $(PROJECT_BUILD_PATH)/src/com/$(APP_COMPANY_NAME)/$(APP_PRODUCT_NAME)
+	mkdir -p $(PROJECT_BUILD_PATH)/lib
+	mkdir -p $(PROJECT_BUILD_PATH)/lib/$(ANDROID_ARCH_NAME)
+	mkdir -p $(PROJECT_BUILD_PATH)/bin
+	mkdir -p $(PROJECT_BUILD_PATH)/res
+	mkdir -p $(PROJECT_BUILD_PATH)/res/drawable-ldpi
+	mkdir -p $(PROJECT_BUILD_PATH)/res/drawable-mdpi
+	mkdir -p $(PROJECT_BUILD_PATH)/res/drawable-hdpi
+	mkdir -p $(PROJECT_BUILD_PATH)/res/values
+	mkdir -p $(PROJECT_BUILD_PATH)/assets
+	mkdir -p $(PROJECT_BUILD_PATH)/assets/$(PROJECT_RESOURCES_PATH)
+	mkdir -p $(PROJECT_BUILD_PATH)/obj/screens
 	$(foreach dir, $(PROJECT_SOURCE_DIRS), $(call create_dir, $(dir)))
 
 define create_dir
-    if not exist $(PROJECT_BUILD_PATH)\obj\$(1) mkdir $(PROJECT_BUILD_PATH)\obj\$(1)
+    mkdir -p $(PROJECT_BUILD_PATH)/obj/$(1)
 endef
     
+ifeq ($(OS),Windows_NT)
+    COPY_COMMAND ?= copy /Y
+    COPY_DIRECTORY_COMMAND = C:\Windows\System32\xcopy /Y /E /F
+    ANDROID_STRING_XML_LINE1 = ^<?xml version="1.0" encoding="utf-8"^?^>
```

Comment:
I would prefer to avoid that many variables, the `ifeq` could just be considered inside the target.

Reply:
I'm fixing this now.

Reply:
Already fixed.